src/components/Product/SingleProduct.tsx

```diff
@@ -0,0 +1,233 @@
+import { faAngleRight, faHeart } from '@fortawesome/free-solid-svg-icons'
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
+import { useEffect, useState } from 'react'
+
+import { Auction } from 'interfaces/Auction'
+import AuctionService from 'services/AuctionService'
+import AuthService from 'services/AuthService'
+
+import './SingleProduct.scss'
+
+const SingleProduct = (props: any) => {
```

Comment:
No need to make huge changes now - but for future reference, if a functionality is not implemented, buttons, etc. that require it should not exist on the UI. There is no feature to leave a review, nor is bidding included in this ticket, so you could have skipped over all of that (which is why the ticket defines out of scope items and assumptions like the assumption that bidding is already implemented in order for it to be shown). The only thing the product overview page should show is the product with the accompanying description that it does have at the time of implementation. Everything else is a potential future add-on. If you aren't sure about what should or shouldn't be included on the UI from the ticket description, you can always ask me.

Reply:
Aha, okaay. For now, I will just delete the review section as it's not implemented yet. And maybe the bidding branch can be first merged with this one to have everything here functional then?

Reply:
Yes, the bidding section should definitely stay, as it's part of a now-finished ticket (but it should have been implemented there instead). The order of merging isn't important as it will all end up in the same place, as in you can either merge this branch into develop and then the bidding branch into develop or merge bidding into this branch and then merge them into develop together.